pkg/tests/tasks/traffic/request_routing_test.go

```diff
@@ -40,41 +45,70 @@ func TestRequestRouting(t *testing.T) {
 		app.InstallAndWaitReady(t, app.Bookinfo(ns))
 
 		productpageURL := app.BookinfoProductPageURL(t, meshNamespace)
-		testUserCookieJar := app.BookinfoLogin(t, meshNamespace)
+		// testUserCookieJar := app.BookinfoLogin(t, meshNamespace)
 
 		t.NewSubTest("not-logged-in").Run(func(t TestHelper) {
 			oc.ApplyString(t, ns, app.BookinfoVirtualServicesAllV1)
 
+			// String value of the Review pod name
+			reviewV1Podname := strings.TrimSpace(shell.Execute(t, fmt.Sprintf(`oc get pods -n %s | grep reviews-v1 | awk '{print $1}'`, ns)))
+			fmt.Println(reviewV1Podname)
+			// Read the html file
+			// Once the html it's on a variable htmlreviewV1
+			templateString, err := os.ReadFile("../../../../testdata/resources/html/productpage-normal-user-v1.html")
+			if err != nil {
+				t.Fatalf("could not read template file %s: %v", templateString, err)
+			}
+
+			htmlFile := template.Run(t, string(templateString), struct{ ReviewV1Podname string }{ReviewV1Podname: reviewV1Podname})
+			fmt.Println(htmlFile)
+
+			// Save the modified HTML content to a local file
+			os.WriteFile("../../../../testdata/resources/html/modified-productpage-user-v1.html", []byte(htmlFile), 0644)
```

Comment:
I think saving the file it's not the best approach to avoid creating new files, you can have the template inside a variable and create a new func that compare but two string

Reply:
without saving, we are not able to compare the files.

Reply:
The good thing it is updating the file. It does not create the new files.